README.md

```diff
@@ -118,6 +118,23 @@ Image name         | Description | Layers | quay.io | docker.io
 
 For each variant there is also available an image with `-alpine` suffix based on Alpine for instance [![Layers info](https://images.microbadger.com/badges/image/radanalyticsio/spark-operator:latest-released-alpine.svg)](https://microbadger.com/images/radanalyticsio/spark-operator:latest-released-alpine)
 
+### Configuring the operator
+
+The spark-operator contains several defaults that are implicit to the creation
+of Spark clusters and applications. Here are a list of environment variables
+that can be set to adjust the default behaviors of the operator.
+
+* `CRD` set to `true` if the operator the operator should respond to Custom
+  Resource Definitions, and set to `false` if it should respone to ConfigMaps.
+* `DEFAULT_SPARK_CLUSTER_IMAGE` a container image reference that will be used
+  for all pods in a `SparkCluster` deployment.
```

Comment:
what about adding: ".. if not specified otherwise in the Custom Resource" or "if the `customImage` isn't set explicitly"

Reply:
that sounds good, i will adjust it